README.md

```diff
@@ -9,3 +9,27 @@ git remote add upstream git@github.com:bitfinexcom/PARENT.git
 git remote -v
 
 git push origin master
+
+## Config validation
+
+If config file has corresponding `.example` file, by default `loadConf` will run a check for missing keys.
+
+If config is missing some keys that are present in `.example` file, program will exit and report all missing keys.
+
+`loadConf` accepts 3rd, optional validation object parameter:
+
+```js
+this.loadConf("cosmos.coin", "coin", {
+  symbol: {
+    required: true,
```

Comment:
This appears to validate if the field is present and does not have a falsy value.
Since we already validate if the field is there by checking the `.example` config, does it make sense to rename to something like `notFalsy` or `notEmpty`?

Reply:
the only scenario I could think of/remember is empty strings for something like hot wallet address that you need to have in config but can't have in example at the time of implementation and that would be covered. Don't remember seeing `0` used in any of values but can extend this check.